<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1065</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="cLayout">
    <item>
     <widget class="QTableView" name="ShowInfotableView"/>
    </item>
    <item>
     <widget class="QStackedWidget" name="messstacked">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_1">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="next_pushButton">
            <property name="text">
             <string>积分计算器</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextEdit" name="textEdit">
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;积分规则&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'宋体'; font-size:10.5pt; color:#000000;&quot;&gt;职称分：教授，&lt;/span&gt;&lt;span style=&quot; font-family:'宋体','Times New Roman'; font-size:10.5pt; color:#000000;&quot;&gt;20&lt;/span&gt;&lt;span style=&quot; font-family:'宋体'; font-size:10.5pt; color:#000000;&quot;&gt;分；副教授，&lt;/span&gt;&lt;span style=&quot; font-family:'宋体','Times New Roman'; font-size:10.5pt; color:#000000;&quot;&gt;16&lt;/span&gt;&lt;span style=&quot; font-family:'宋体'; font-size:10.5pt; color:#000000;&quot;&gt;分；讲师，&lt;/span&gt;&lt;span style=&quot; font-family:'宋体','Times New Roman'; font-size:10.5pt; color:#000000;&quot;&gt;12&lt;/span&gt;&lt;span style=&quot; font-family:'宋体'; font-size:10.5pt; color:#000000;&quot;&gt;分；助教，&lt;/span&gt;&lt;span style=&quot; font-family:'宋体','Times New Roman'; font-size:10.5pt; color:#000000;&quot;&gt;8&lt;/span&gt;&lt;span style=&quot; font-family:'宋体'; font-size:10.5pt; color:#000000;&quot;&gt;分。&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'宋体'; font-size:10.5pt; color:#000000;&quot;&gt;工龄分：每年&lt;/span&gt;&lt;span style=&quot; font-family:'宋体','Times New Roman'; font-size:10.5pt; color:#000000;&quot;&gt;0.5&lt;/span&gt;&lt;span style=&quot; font-family:'宋体'; font-size:10.5pt; color:#000000;&quot;&gt;分；每月&lt;/span&gt;&lt;span style=&quot; font-family:'宋体','Times New Roman'; font-size:10.5pt; color:#000000;&quot;&gt;0.0416&lt;/span&gt;&lt;span style=&quot; font-family:'宋体'; font-size:10.5pt; color:#000000;&quot;&gt;分。&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'宋体'; font-size:10.5pt; color:#000000;&quot;&gt;年龄分：每岁&lt;/span&gt;&lt;span style=&quot; font-family:'宋体','Times New Roman'; font-size:10.5pt; color:#000000;&quot;&gt;0.1&lt;/span&gt;&lt;span style=&quot; font-family:'宋体'; font-size:10.5pt; color:#000000;&quot;&gt;分；每月&lt;/span&gt;&lt;span style=&quot; font-family:'宋体','Times New Roman'; font-size:10.5pt; color:#000000;&quot;&gt;0.0083&lt;/span&gt;&lt;span style=&quot; font-family:'宋体'; font-size:10.5pt; color:#000000;&quot;&gt;分。&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'宋体'; font-size:10.5pt; color:#000000;&quot;&gt;学龄分：博士，&lt;/span&gt;&lt;span style=&quot; font-family:'宋体','Times New Roman'; font-size:10.5pt; color:#000000;&quot;&gt;4.5&lt;/span&gt;&lt;span style=&quot; font-family:'宋体'; font-size:10.5pt; color:#000000;&quot;&gt;分；硕士，&lt;/span&gt;&lt;span style=&quot; font-family:'宋体','Times New Roman'; font-size:10.5pt; color:#000000;&quot;&gt;3.25&lt;/span&gt;&lt;span style=&quot; font-family:'宋体'; font-size:10.5pt; color:#000000;&quot;&gt;分，本科，&lt;/span&gt;&lt;span style=&quot; font-family:'宋体','Times New Roman'; font-size:10.5pt; color:#000000;&quot;&gt;2&lt;/span&gt;&lt;span style=&quot; font-family:'宋体'; font-size:10.5pt; color:#000000;&quot;&gt;分；&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'宋体'; font-size:10.5pt; color:#000000;&quot;&gt;现住房分：每平米&lt;/span&gt;&lt;span style=&quot; font-family:'宋体','Times New Roman'; font-size:10.5pt; color:#000000;&quot;&gt;0.1&lt;/span&gt;&lt;span style=&quot; font-family:'宋体'; font-size:10.5pt; color:#000000;&quot;&gt;分；&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'宋体'; font-size:10.5pt; color:#000000;&quot;&gt;家庭成员分：每人&lt;/span&gt;&lt;span style=&quot; font-family:'宋体','Times New Roman'; font-size:10.5pt; color:#000000;&quot;&gt;2&lt;/span&gt;&lt;span style=&quot; font-family:'宋体'; font-size:10.5pt; color:#000000;&quot;&gt;分；&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'宋体'; font-size:10.5pt; color:#000000;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <widget class="QWidget" name="horizontalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>90</y>
          <width>232</width>
          <height>41</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>入职时间</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="hireDate_dateEdit"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>170</y>
          <width>184</width>
          <height>31</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>年 龄</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="age_spinBox"/>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>岁</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget_4">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>240</y>
          <width>180</width>
          <height>31</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <spacer name="horizontalSpacer_11">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>学 位</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="degree_comboBox">
           <property name="currentText">
            <string>高中</string>
           </property>
           <item>
            <property name="text">
             <string>高中</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>本科</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>硕士</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>博士</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_12">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget_5">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>240</y>
          <width>192</width>
          <height>31</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>职 称</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="position_comboBox">
           <item>
            <property name="text">
             <string>教授</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>副教授</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>讲师</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>助教</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget_6">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>310</y>
          <width>244</width>
          <height>22</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <spacer name="horizontalSpacer_9">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>住房面积</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="area_doubleSpinBox"/>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>平方米</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_10">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget_7">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>20</y>
          <width>123</width>
          <height>25</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QPushButton" name="last_pushButton">
           <property name="text">
            <string>积分细则</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget_8">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>170</y>
          <width>172</width>
          <height>31</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <spacer name="horizontalSpacer_13">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>成员数</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_14">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget_9">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>350</y>
          <width>169</width>
          <height>25</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <spacer name="horizontalSpacer_15">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="caculatepushButton">
           <property name="text">
            <string>开始计算</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_16">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="score_label">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>390</y>
          <width>391</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="messframe">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>40</number>
       </property>
       <item>
        <widget class="QLabel" name="messLabel">
         <property name="text">
          <string>messlabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1065</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="actionFileOpen"/>
    <addaction name="separator"/>
    <addaction name="actionFileSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>账户(&amp;T)</string>
    </property>
    <addaction name="actionAccontIn"/>
    <addaction name="actionAcoontOut"/>
   </widget>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>查询(&amp;F)</string>
    </property>
    <addaction name="actionFindEmployee"/>
    <addaction name="actionScore"/>
    <addaction name="action"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="actionabout"/>
   </widget>
   <widget class="QMenu" name="menu_Y">
    <property name="title">
     <string>添加(&amp;Y)</string>
    </property>
    <addaction name="actionAddEmployee"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_F"/>
   <addaction name="menu_Y"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionShowMessage"/>
   <addaction name="separator"/>
   <addaction name="action"/>
   <addaction name="separator"/>
   <addaction name="actionFindEmployee"/>
   <addaction name="separator"/>
   <addaction name="actionAddEmployee"/>
   <addaction name="action_2"/>
   <addaction name="separator"/>
   <addaction name="actionScore"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
  </widget>
  <action name="actionFileOpen">
   <property name="text">
    <string>打开文件(&amp;O)</string>
   </property>
  </action>
  <action name="actionFileSave">
   <property name="text">
    <string>保存文件(&amp;S)</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出(&amp;X)</string>
   </property>
  </action>
  <action name="actionAccontIn">
   <property name="text">
    <string>登入账户</string>
   </property>
  </action>
  <action name="actionAcoontOut">
   <property name="text">
    <string>退出账户</string>
   </property>
  </action>
  <action name="actionFindEmployee">
   <property name="text">
    <string>查找职工</string>
   </property>
  </action>
  <action name="actionabout">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="actionShowMessage">
   <property name="text">
    <string>显示信息</string>
   </property>
   <property name="toolTip">
    <string>显示信息</string>
   </property>
  </action>
  <action name="actionScore">
   <property name="text">
    <string>积分情况</string>
   </property>
  </action>
  <action name="actionAddEmployee">
   <property name="text">
    <string>添加职工</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>查找职工家属</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>删除职工</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
